mlrun/feature_store/feature_vector.py

```diff
@@ -336,7 +338,10 @@ def add_feature(name, alias, feature_set_object, feature_set_full_name):
             feature_fields = feature_set_object.spec.features.keys()
             if feature_name == "*":
                 for field in feature_fields:
-                    if field != feature_set_object.spec.timestamp_key:
+                    if (
+                        field != feature_set_object.spec.timestamp_key
+                        and field != label_column_name
```

Comment:
As mentioned in previous comment - also verify it's the same feature-set as the one where the `label_column` comes from.